scripts/pre_push_hook.py

```diff
@@ -214,6 +214,7 @@ def _collect_files_being_pushed(ref_list, remote):
 
 
 def _get_refs():
+    """Returns the ref list taken from STDIN."""
```

Comment:
Does this need a returns section?

Reply:
Here adding Args and Returns to test and script files isn't mandatory. Even we're adding docstrings as we could not find good way to ignore these files just for docstrings test.